src/main/kotlin/com/exactpro/th2/common/schema/message/impl/rabbitmq/transport/ParsedMessage.kt

```diff
@@ -102,7 +99,6 @@ class ParsedMessage private constructor(
     interface FromMapBuilder : Builder<FromMapBuilder> {
         fun setBody(body: Map<String, Any?>): FromMapBuilder
         fun bodyBuilder(): MapBuilder<String, Any?>
-        fun addField(name: String, value: Any?): FromMapBuilder
```

Comment:
Why have you decided to remove that method? It allows adding a field without breaking the methods chain

Reply:
This method can confuse users when they use `.apply {` approach to fill message. I have add a separate extension method for that if you want work with builder directly.

Reply:
They can't use the `apply` method in Java... The extension method you added can be used conveniently only from Kotlin. In Java it will become a static method

Reply:
Reverted